src/sentry/integrations/discord/requests/base.py

```diff
@@ -152,16 +132,41 @@ def get_identity_user(self) -> RpcUser | None:
 
     def get_identity(self) -> RpcIdentity | None:
         if not self._identity:
+            logger.info(
+                "discord.validate.identity.none",
+                extra={
```

Comment:
@17hogeju I would define a variable for `extra` to re-use it for the different `logger.info` calls